docs/pages/modules/third-party-library.mdx

```diff
@@ -19,6 +19,10 @@ This tutorial focuses on utilizing the Expo Modules API to create radial charts
 The iOS library is inspired by the Android library, so they both have a very similar API and functionality.
 This makes them a good example for this tutorial.
 
+## How to wrap native libraries video tutorial
```

Comment:
```suggestion
## Wrapping third-party native libraries video tutorial
```

I'd like us to find a different pattern for these video section titles (or maybe present video sections as a whole), especially in guides where other section titles (the majority of H2s) are labeled as steps. Visually, it is a bit confusing that this section ends abruptly, and then the steps start, and there's no distinction between them.